package.json

```diff
@@ -62,8 +63,7 @@
     "web3-typescript-typings": "^0.9.8",
     "web3-utils": "^1.0.0-beta.29",
     "when": "^3.7.8",
-    "yargs": "^11.0.0",
-    "zeppelin-solidity": "^1.6.0"
+    "yargs": "^11.0.0"
```

Comment:
do we need yargs ?

Reply:
not anymore actually, since https://github.com/daostack/arc/pull/347

Reply:
updated: https://github.com/daostack/arc/pull/357/commits/3acbe39c04b10f6f5699e96068865757c15aaea7